app/builders/linked_account_email_builder.rb

```diff
@@ -52,7 +50,7 @@ def bulleted_active_subscriptions
   end
 
   def sign_in_url
-    GovukPersonalisation::Urls.sign_in
+    "https://www.gov.uk/account"
```

Comment:
Humm, we have had cause before to see the output of the email - I think in a log or /tmp text format.
This would make testing things harder on local, as well as not possible to follow links through from integration and staging, wouldn't it?